app/views/contexts/hub_context.py

```diff
@@ -171,7 +185,8 @@ def _get_rows(self, enabled_section_ids) -> List[Mapping[str, Union[str, List]]]
                             )
                         )
                 else:
-                    rows.append(self._get_row_for_section(section_title, section_id))
+                    # Type ignore: section_title has to exist by this step
+                    rows.append(self._get_row_for_section(section_title, section_id))  # type: ignore
```

Comment:
This is a valid error, section title might be `None` therefore you need to update `_get_row_for_section` so that the title is an optional str.

Reply:
👍

Reply:
The type ignore is still there.

Reply:
👍

Reply:
Still there on line 194